src/Components/CreateImageWizard/steps/imageName.js

```diff
@@ -20,15 +20,18 @@ export default {
       name: 'plain-text-component',
       label: (
         <p>
-          Optionally enter a name for your image. All images will have a UUID.
+          Optionally enter a name and description to identify your image later
```

Comment:
Let's remove "and description" for now as we are not adding the text input for Description in this commit.